addons/skin.estuary/xml/Includes_DialogSelect.xml

```diff
@@ -624,15 +624,15 @@
 			<control type="list" id="6">
 				<left>20</left>
 				<top>100</top>
-				<width>880</width>
+				<width>930</width>
 				<bottom>20</bottom>
 				<onup>6</onup>
 				<ondown>6</ondown>
 				<onleft>9001</onleft>
 				<onright>61</onright>
 				<pagecontrol>61</pagecontrol>
 				<scrolltime>200</scrolltime>
-				<itemlayout height="75" width="880">
+				<itemlayout height="75" width="930">
```

Comment:
Same as above.

Reply:
Changed that should have been reverted, fixed.